inst/templates/gitbook.html

```diff
@@ -123,12 +123,22 @@ <h2 class="subtitle"><em>$subtitle$</em></h2>
 $endif$
 $for(author)$
 $if(author.name)$
+<style type="text/css">
+  #header p.author {
+    margin-top: 1em;
+    margin-bottom: 0em;
```

Comment:
Just a cosmetic change:

```suggestion
    margin: 1em 0 0 0;
```